maintainers/maintainer-list.nix

```diff
@@ -10186,6 +10186,12 @@
     email = "vq@erq.se";
     name = "Daniel Nilsson";
   };
+  vrinek = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add vrinek``.

Reply:
Resolved in https://github.com/NixOS/nixpkgs/pull/117110/commits/c26a559134f8dc524e8cc057e2cd427143ed104e.